M/MFEM/build_tarballs.jl

```diff
@@ -0,0 +1,50 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "MFEM_seq"
+version = v"4.4.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/mfem/mfem.git", "a1f6902ed72552f3e680d1489f1aa6ade2e0d3b2")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd mfem*
+mkdir build; cd build
+cmake -DCMAKE_INSTALL_PREFIX=${prefix}       -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}       -DCMAKE_BUILD_TYPE=Release       -DBUILD_SHARED_LIBS=1       ..
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("x86_64", "macos"; ),
+    Platform("aarch64", "macos"; ),
+    Platform("x86_64", "freebsd"; ),
+    Platform("i686", "windows"; ),
+    Platform("x86_64", "windows"; )
+]
```

Comment:
Maybe we can build for all platforms and see what's the problem with the failing ones?
```suggestion
platforms = supported_platforms()
```